sycl/test-e2e/AddressSanitizer/out-of-bounds/USM/matrix_multiply.cpp

```diff
@@ -0,0 +1,105 @@
+// REQUIRES: linux
+// RUN: %{build} %device_sanitizer_flags -DMALLOC_DEVICE -O1 -g -o %t
+// RUN: %{run} %t &> %t.txt ; FileCheck --check-prefixes CHECK,CHECK-DEVICE --input-file %t.txt %s
```

Comment:
Suggest to run the command with 'not' tool. Then you don't need to re-direct the output to a temporary txt file.

Reply:
done